model/metrics/http.yaml

```diff
@@ -31,6 +31,25 @@ groups:
             if it's sent in absolute-form.
           - Port identifier of the `Host` header
       # todo (lmolkova) build tools don't populate grandparent attributes
+      - ref: error.id
```

Comment:
I guess we need to repeat this because of https://github.com/open-telemetry/build-tools/issues/133, right?

Reply:
yep :(

Reply:
We should really fix that.. I wish I had the time to do it :(